src/app/Http/Controllers/QuizController.php

```diff
@@ -2,46 +2,32 @@
 
 namespace App\Http\Controllers;
 
+use App\BigQuestion;
+use App\Question;
+use App\Choice;
 use Illuminate\Http\Request;
-use Illuminate\Support\Facades\DB;
 
 
 class QuizController extends Controller
 {
 
     // 問題リスト表示用
-    public function index_links(Request $request) {
+    public function list(Request $request) 
+    {
         // big_questions テーブルのデータを全て取得し $links に格納
-        $links = DB::table('bigquestions')->get();
+        // $links = DB::table('bigquestions')->get(); 
+        $links = BigQuestion::all();
```

Comment:
BigQuestionのリストなので$linksだと変数名的にわかりにくい可能性があるのでモデル名の複数形などが良さそうです
```suggestion
        $bigQuestions = BigQuestion::all();
```